asmcli/asmcli.sh

```diff
@@ -167,7 +171,8 @@ init() {
   readonly RELEASE; readonly RELEASE_LINE; readonly PREVIOUS_RELEASE_LINE; readonly KPT_BRANCH;
 
   KPT_URL="https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages"
-  KPT_URL="${KPT_URL}.git/asm@${KPT_BRANCH}"; readonly KPT_URL;
+  # KPT_URL="${KPT_URL}.git/asm@${KPT_BRANCH}"; readonly KPT_URL;
+  KPT_URL="${KPT_URL}.git/asm@v2"; readonly KPT_URL;
```

Comment:
I updated the trigger to pass in the right variable here, you shouldn't need to hardcode it.

Reply:
sweet